spec/src/main/asciidoc/entity.asciidoc

```diff
@@ -432,6 +432,61 @@ The following subinterfaces of `Attribute` are recommended to obtain the full be
 - `NavigableAttribute` for entity attributes that are embeddables or associations.
 - `BasicAttribute` for other types of entity attributes, such as collections.
 
+To assist developers in selecting the appropriate subinterface of `Attribute` when declaring static metamodel fields, the following table maps common Java types to the corresponding attribute interface. While multiple Java types may be compatible with the same subinterface, this guidance ensures consistent and type-safe usage across different entity models.
+
+[cols="2,4", options="header"]
+|===
+| Java Type(s) | Attribute Interface
+
+| `String`, any `CharSequence`
+| `TextAttribute`
+
+
+| Primitive numbers (`byte`, `int`, `double`, etc.)  and any subclass of `java.lang.Number` (e.g., `BigDecimal`, `BigInteger`)
+| `NumericAttribute`
+
+| `LocalDate`, `LocalDateTime`, `LocalTime`, `Instant`, `Year`, `YearMonth`
```

Comment:
If including `YearMonth`, then `MonthDay` should also be included,
```suggestion
| `LocalDate`, `LocalDateTime`, `LocalTime`, `Instant`, `MonthDay`, `Year`, `YearMonth`
```
Does anyone know if JPA supports EXTRACT from a YearMonth and MonthDay or is planning to add that support?  That would be the main advantage I see to including them here.